lib/LicenseTermsList/readme.md

```diff
@@ -0,0 +1,17 @@
+# LicenseTermsList
+
+
+
+
+## Basic Usage
+Simply passing a license object and it's terms is all that's required.
```

Comment:
Can you provide provide more info about the `terms` array and its structure in the **Basic Usage** and **Props** sections? I try to write these readmes so it's possible to reference the readme rather than having to look at the code when using the components, and I'd have no idea what the terms array is supposed to look like from this.

(And also remove the extra blank lines after the header?)